src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -5049,6 +5049,7 @@ Note* MusicXMLParserPass2::note(const QString& partId,
     DirectionV stemDir = DirectionV::AUTO;
     bool noStem = false;
     NoteHeadGroup headGroup = NoteHeadGroup::HEAD_NORMAL;
+    bool headInvisible = false;
```

Comment:
in 3.x this is called `hasHead` (and is one line above) and defaults to `true`.
Might make sense to keep naming in scync.